processor/clients/integration_client.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import json
+import logging
+
+log = logging.getLogger()
+
+class Integration:
+    def __init__(self, id, application_id, dataset_id, package_ids, params):
+        self.id = id
+        self.application_id = application_id
+        self.dataset_id = dataset_id
+        self.package_ids = package_ids
+        self.params = params
+
+class IntegrationClient:
+    def __init__(self, api_host):
+        self.api_host = api_host
+
+    # NOTE: integration API currently returns a 200 response
+    #       with an empty body even when an integration does not exist
+    def get_integration(self, session_token, integration_id):
+        url = f"{self.api_host}/integrations/{integration_id}"
```

Comment:
@rohanshah - sorry I had missed this yesterday. But I marked the integrations endpoint as deprecated, so please use this endpoint instead: https://docs.pennsieve.io/reference/getworkflowinstance.

Transitioning into workflow terminology now, so if you'd like you can update your functions to follow the same concept as well.